packages/next-swc/crates/core/src/lib.rs

```diff
@@ -53,6 +53,7 @@ pub mod next_dynamic;
 pub mod next_ssg;
 pub mod page_config;
 pub mod react_remove_properties;
+pub mod relay;
```

Comment:
isn't `#[cfg]` for was required?

Reply:
Thanks @kdy1!

@tbezman what was the reason you've disabled `relay` on wasm? Was it because of the build issues with all these extra dependencies on the `relay-compiler`? Or was it something more fundamental?

IMO, if this was added to fix build issues, I don't think we need this anymore, as the current implementation doesn't have these extra dependencies.

Reply:
According to @timneutkens:
> the wasm binary is used in Next.js live which doesn't support Relay as it'd also require the Relay compiler to run (it can't run in the browser) so the wasm binary can't hold filesystem operations

https://github.com/vercel/next.js/pull/33240#issuecomment-1017317532

Reply:
@wongmjane thanks! 

I think we're looking into ways of running `relay-compiler` in the browser. One live example is this https://relay.dev/compiler-explorer/ - it runs a special version of compiler, compiled to wasm, and we could theoretically use this as a foundation for the full-featured relay compiler in `wasm`. But for now, yes - I'll send a PR to restore these `cfg` changes.